Game of Game of Life.py

```diff
@@ -0,0 +1,110 @@
+### a,b refers to cell (a,b) whereas x,y refers to pixel coordinates ###
+
+import pygame as pg, math as m, time as t, random as r
+
+Width=50    #C 50 How many squares wide the board is
```

Comment:
every constant here should be removed and added to config file #1

Reply:
done